packages/medusa/src/api/routes/admin/publishable-api-keys/index.ts

```diff
@@ -80,14 +80,61 @@ export default (app) => {
   )
 }
 
+/**
+ * @schema AdminPublishableApiKeysRes
+ * type: object
+ * properties:
+ *   publishable_api_key:
+ *     $ref: "#/components/schemas/PublishableApiKey"
+ */
 export type AdminPublishableApiKeysRes = {
   publishable_api_key: PublishableApiKey
 }
+
+/**
+ * @schema AdminPublishableApiKeysListRes
+ * type: object
+ * properties:
+ *   publishable_api_keys:
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/PublishableApiKey"
+ */
 export type AdminPublishableApiKeysListRes = PaginatedResponse & {
   publishable_api_keys: PublishableApiKey[]
 }
+
+/**
+ * @schema AdminPublishableApiKeyDeleteRes
+ * type: object
+ * properties:
+ *   id:
+ *     type: string
+ *     description: The ID of the deleted PublishableApiKey.
+ *   object:
+ *     type: string
+ *     description: The type of the object that was deleted.
+ *     default: publishable_api_key
+ *   deleted:
+ *     type: boolean
+ *     description: Whether the PublishableApiKeys was deleted.
+ *     default: true
+ */
 export type AdminPublishableApiKeyDeleteRes = DeleteResponse
 
+/**
+ * @schema AdminPublishableApiKeysListSalesChannelsRes
+ * type: object
+ * properties:
+ *   sales_channels:
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/SalesChannel"
+ */
+export type AdminPublishableApiKeysListSalesChannelsRes = {
+  sales_channels: SalesChannel[]
+}
```

Comment:
Missing declaration for list-publishable-api-key-sales-channels. JS client wrongfully type listSalesChannels response as `AdminSalesChannelsListRes`. See [file](AdminPublishableApiKeysListSalesChannelsRes).